index.js

```diff
@@ -109,7 +92,7 @@ function hasCauseInstanceOf (error, klass) {
   return error instanceof klass || hasCauseInstanceOf(error.cause, klass)
 }
 
-exports.Error = ErrorTypeError
```

Comment:
I think we could do `module.exports = OError`. The require would look a bit non standard (unless/until we change the package name to `@overleaf/o-error` or similar), but it would make it cleaner I think.

```js
const OError = require('overleaf-error-type')

OError.getFullInfo(...)
OError.getFullStack(...)
```

It would also mean we could make the helpers static methods in `OError`, which seems reasonable.

(And we could just assign the legacy methods to OError, as in `OError.defineIn = ...` to minimise changes there.)

WDYT?

Reply:
I would vote for that. I think the plan is to remove the legacy methods in the near future and the proposed method is more standard (I think) for a single export.

Reply:
Yep, good points